src/main/webroot/upload/csv/index.html

```diff
@@ -51,8 +51,8 @@ <h1 id="csv-upload-title">CSV Upload</h1>
             <div class="form-group">
               <label for="iiif-version">IIIF manifest version:</label> 
 	          <select name="iiif-version" id="iiif-version" required>
-	              <option value=""/>
-                  <option value="v2">v2</option>
+	              <!--option value=""/-->
```

Comment:
Why not delete instead of comment out?

Reply:
IIRC, the blank option was there to encourage user to think about what IIIF manifest version they wanted built; I'm guessing we'll want to go back to that approach once the v3 issue is solved.  If I'm wrong here, I'll delete the blank line.

Reply:
Ah, okay, I see. I was thinking we'd want to make v3 the default once the client issue is resolved. @markmatney do you have opinions?

Reply:
I could see it either way. I do kind of like the user having to make a conscious choice, but a default value might reduce the likelihood of user error (where that default could perhaps be configured by an environment variable depending on the particular instance of Fester (each of which might have a different expected IIIF client) that the user is ingesting into).

FWIW, this could be implemented as two radio inputs where the user would still have to choose one in order to submit the form.

Reply:
Geno said he could see v3 being the default (though didn't sound very opinionated about it). Waiting on Lisa's response.

I do like the idea of the radio buttons. They can start out unselected and then they give an either/or choice.

Reply:
Lisa preferred a default value (v2 now and v3 in the future). I'd say let's remove the commented out blank value and give the radio buttons a try.

Reply:
Switched to radio buttons.